recipes/freva/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "freva" %}
+{% set version = "2206.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/freva-{{ version }}.tar.gz
+  sha256: aceb6128ac41d01ae4a84ba7336ef8bbcb66f82ff3f31df75d5b61f0c045c5df
+
+build:
+  entry_points:
+    - freva = freva.cli:main
+    - freva-databrowser = freva.cli.databrowser:main
+    - freva-esgf = freva.cli.esgf:main
+    - freva-history = freva.cli.history:main
+    - freva-plugin = freva.cli.plugin:main
+    - freva-crawl-my-data = freva.cli.crawl_my_data:main
+  skip: true  # [win]
```

Comment:
What is the issue on Windows? Missing dependency? If so I believe we can still make this `noarch`.

Reply:
Thanks for taking the time to look at this. I believe the problem is with the [mysqlclient](https://pypi.org/project/mysqlclient/) package. Because the package docs say: "Building mysqlclient on Windows is very hard." and because I ran into [a dependency error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=540730&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=804): `conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {MatchSpec("mysqlclient")}` I decided to drop windows support.

But if you think we can still make this noarch or even better find a way of integrating windows even better. Do you have something in mind?

Reply:
We usually make pkgs noarch and let the solver "fail" to install on Windows. Noarch pkgs saves CI time and are easier to maintain b/c they require less rebuilds.

Reply:
Ok, then I'll re-add `noarch`.